moonraker/components/file_manager/metadata.py

```diff
@@ -740,7 +740,7 @@ def parse_first_layer_bed_temp(self) -> Optional[float]:
 
     def parse_nozzle_diameter(self) -> Optional[float]:
         return _regex_find_first(
-            r";Nozzle\sdiameter\s=\s(\d+\.\d*)", self.header_data)
+            r";Dimension:(\s\d+\.\d+)*", self.header_data)
```

Comment:
While this will work, it could potentially match a value that doesn't strictly refer to the nozzle diameter.  I would recommend a more strict regex, such as:
```
r";Dimension:(?:\s\d+\.\d+){3}\s(\d+\.\d+)"
```

Reply:
Thanks! Yes this more strict regex might be better. I force pushed a fix to this PR.